packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -893,6 +893,13 @@ export default function getConfigFileMigrationsFactory(homeDir, defaultConfigs)
           });
         return configFile;
       },
+      '1.3.0': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([, options]) => {
+            options.platform.drive.tenderdash.docker.image = base.get('platform.drive.tenderdash.docker.image');
```

Comment:
It's more reliable to use hardcoded params, otherwise future changes of the base config structure will affect migrations.

Reply:
changed to 1.3

Reply:
hardcoded `dashpay/tenderdash:1`